src/governance/object.h

```diff
@@ -223,16 +223,16 @@ class CGovernanceObject
 
     // CORE OBJECT FUNCTIONS
 
-    bool IsValidLocally(std::string& strError, bool fCheckCollateral) const EXCLUSIVE_LOCKS_REQUIRED(cs_main);
+    bool IsValidLocally(CDeterministicMNManager& dmnman, std::string& strError, bool fCheckCollateral) const EXCLUSIVE_LOCKS_REQUIRED(cs_main);
```

Comment:
I think it makes sense to pass in all CGovernanceObject's methods an MNList (result of call `deterministicMNManager->GetListAtChainTip();`) rather than CDeterministicMNManager.

CDeterministicMNManager is little over-kill

Reply:
I like the idea. Working on it...